README.md

```diff
@@ -6,6 +6,42 @@
 Visualize your digital footprint.
 
 ## Installing Excavator
+There are different ways to install Excavator. The first method is particularly meant to be used by non-developers users.
+
+### Install the binary 
+If you don't have Rust installed on your computer, this method is for you.
+First click on "Releases" :
+
+![](/img/img-1-install.png)
+
+You should see the last release on your screen (v0.2.0 when we are writing this).
+Depending on your OS (Windows, Linux or Mac), click on one the three files highlighted.
+
+![](/img/img-2-install.png)
+
+It should download the file on your device. Then you need to uncompress it.
+
+#### Windows 
+We are using 7zip to unzip the file. 
+
+![](/img/img-3-install.png)
+
+Put this file in `C:\Windows\System32`, and the installation is completed. You should now be able to use Excavator in a terminal. 
+
+#### Linux
+Click on the file you just downloaded.
+![](/img/img-4-install.png)
+
+Then extract it when you want. In our case, we extract it in `Desktop`.
+
+![](/img/img-5-install.png)
+
+Finally, open a terminal and type the following command `sudo cp PATH_TO_EXCAVATOR /usr/bin` (replace PATH_TO_EXCAVATOR by the path to where you extracted excavator in the previous step). 
```

Comment:
I think there are multiple possible places were to place a file. Maybe `/usr/local/bin` would better on linux, because `/usr/bin` is for files provided by the OS.
There are many options: https://unix.stackexchange.com/questions/36871/where-should-a-local-executable-be-placed

However, it would be also OK to just run `excavator` from any directory. It may just need to be made executable.

Reply:
You are right, `/usr/local/bin` is definitely better for Linux. 
Besides, I think that the same path exists for Mac, maybe can you confirm this?